tests/unittests/GradCheckTest.cpp

```diff
@@ -29,7 +29,14 @@ using llvm::cast;
 
 class GradCheckBase : public ::testing::TestWithParam<std::string> {
 public:
-  ExecutionEngine EE_{GetParam()};
+  ExecutionEngine2 EE_{GetParam()};
```

Comment:
Looks like you only use `EE_` in one place, `gradientCheckCrossEntropyLoss`, so I'd be inclined to remove this and just use `EET_` there